onboard/catkin_ws/src/motion_planning/scripts/competition_task.py

```diff
@@ -1,7 +1,7 @@
 from combination_tasks import ListTask
 from move_tasks import MoveToPoseGlobalTask
 from task import Task
-from prequal_tasks import PreQualGlobalTask
+# from prequal_tasks import PreQualGlobalTask
```

Comment:
Since it doesn't seem like you're using this
```suggestion

```

Reply:
when we do prequal, we'll need it, so I wanted to leave it for convenience. Alright with you?